nightshade-webclients.yml

```diff
@@ -145,8 +119,13 @@
 
   vars:
     omero_web_config_set: "{{ omero_web_config_set_for_group | combine(omero_web_config_set_for_host) }}"
-
-    nginx_version: 1.14.2
+    omero_figure_release: "{{ omero_figure_release_override | default('4.0.2') }}"
+    omero_fpbioimage_release: "{{ omero_fpbioimage_release_override | default('0.3.0') }}"
+    omero_iviewer_release: "{{ omero_iviewer_release_override | default('0.7.0') }}"
+    omero_parade_release: "{{ omero_parade_release_override | default('0.1.2') }}"
+    omero_webtagging_autotag_release: "{{ omero_webtagging_autotag_release_override | default('3.0.2') }}"
+    omero_webtagging_tagsearch_release: "{{ omero_webtagging_tagsearch_release_override | default('3.0.3') }}"
+    omero_web_release_version: "{{ omero_web_release_version_override | default('5.4.10') }}"
```

Comment:
Are you planning to upgrade Nightshade OMERO.web to 5.4.10, and the web-apps too?
Current versions on ns-web:
OMERO.py: 5.4.9
```
$ /opt/omero/web/venv/bin/pip list |grep omero
omero-figure (4.0.0)
omero-fpbioimage (0.3.0)
omero-iviewer (0.6.0)
omero-marshal (0.5.3)
omero-parade (0.1.2)
omero-webtagging-autotag (3.0.2)
omero-webtagging-tagsearch (3.0.3)
```

Reply:
Everything needs to be upgraded. apps and web
If people are ok with change, I will go directly to 5.5.0 for web

Reply:
I think it's important that when a PR is merged it represents the current state of our systems, so depending on your preference either change the versions to match what's currently deployed, or include the 5.5.0 upgrade in this PR.

Reply:
I have only run in ``checkmode`` so nothing has been upgraded. 
In my view, It is preferable to have any changes reviewed before running without ``checkmode`` if possible

Reply:
Agreed. With nightshade it's more difficult, but I understood the general agreement to be:

 * Open a PR with checkmode against staging & prod server output
 * If general agreement, run against staging unmerged
 * If everything looks ok, merge and then rollout to prod